app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] Kindly add users_controller  #index and #show actions.
````ruby
 
  def index; end

  def show; end
````
- It's okay for the actions to be empty for now 👍🏿